project/core/urls.py

```diff
@@ -22,6 +22,7 @@
 media_urlpatterns = static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
 
 urlpatterns = [
+    path('__debug__/', include('debug_toolbar.urls')),
```

Comment:
Use double quotes to align with the `black` formatting.
```suggestion
    path("__debug__/", include("debug_toolbar.urls")),
```

Reply:
currently, these all are with double quotes.